client/signup/main.jsx

```diff
@@ -255,7 +259,11 @@ class Signup extends Component {
 		debug( 'Signup component mounted' );
 		this.props.flowName === 'onboarding' && ! this.props.isLoggedIn && addHotJarScript();
 		this.startTrackingForBusinessSite();
-		recordSignupStart( this.props.flowName, this.props.refParameter, this.getRecordProps() );
+
+		if ( ! isNewsletterFlow( this.props.flowName ) ) {
```

Comment:
**NOTE:**

The newsletter flow switches between the new `stepper` framework and the traditional domains and plans selection screen.

The `calypso_signup_start` tracks event is already called by the stepper flow when newsletter onboarding is initially loaded, so it's unnecessary for it to be called in traditional signup too